app/serializers/api/v1/data/ndc_sdg/ndc_sdg_serializer.rb

```diff
@@ -4,19 +4,15 @@ module Data
       module NdcSdg
         class NdcSdgSerializer < ActiveModel::Serializer
           attribute :id
-          attribute(:country) { object['country'] }
           attribute(:iso_code3) { object['iso_code3'] }
+          attribute(:country) { object['country'] }
+          attribute(:sdg) { object['sdg'] }
+          attribute(:sdg_target) { object['sdg_target'] }
           attribute(:indc_text) { object['indc_text'] }
           attribute(:status) { object['status'] }
+          attribute(:sector) { object['sector'] }
           attribute(:climate_response) { object['climate_response'] }
           attribute(:type_of_information) { object['type_of_information'] }
-          attribute(:sector) { object['sector'] }
-          attribute(:target_number) { object['target_number'] }
-          attribute(:target) { object['target'] }
-          attribute(:goal_number) { object['goal_number'] }
-          attribute(:goal) { object['goal'] }
-          attribute(:document_type) { object["document_type"] }
```

Comment:
We need the document_type and language in the table to be able to redirect to the correct indc_text document. Maybe include it only in the table data JSON and not the CSV and it will be filtered so it won't appear in the frontend table